client/state/selectors/is-vip-site.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if the site is VIP
+ *
+ * If the site is missing returns null.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether site is VIP
+ */
+export default function isVipSite( state, siteId ) {
+	const site = getRawSite( state, siteId );
+	if ( ! site || typeof site.is_vip === 'undefined' ) {
```

Comment:
If we just need to test existence, maybe `site.hasOwnProperty( 'is_vip' )`. Is this value never false?